_bmad-output/planning-artifacts/architecture.md

```
@@ -0,0 +1,635 @@
+- Existing `OMNI_TRACER_API` package → extend with new procedures
+- Existing `SubscriberService` → add procedure generation on registration
+- Existing Settings UI → add danger zone options
+- Existing `TracerService` → remains unchanged (receiving logic unchanged)
+
+---
+
+## Project Structure & Boundaries
+
+### Complete Project Directory Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)